docs/elements.md

```diff
@@ -347,6 +350,75 @@ Attribute | Type | Default | Description
 
 -----
 
+### `pl-order-blocks` element
```

Comment:
I'd prefer all the long lines in the docs to be wrapped at 80 or 100 characters, but that's up to @mwest1066

Reply:
It looks like we aren't consistently wrapping elsewhere is this file, so I'm happy to leave this as is for now.